esmvaltool/recipes/mpqb/recipe_mpqb_test_BM.yml

```diff
@@ -21,6 +21,10 @@ timeperiod: &timeperiod
 #  - {dataset: CDS-UERRA, type: reanaly, project: OBS6, mip: E6hr,
 #     version: UERRA-HARMONIE, tier: 3, frequency: 6hr, <<: *timeperiod}
 
+mmm_stat: &mmm_stat
```

Comment:
Adds the multimodel-mean and -median statistic to the preprocessor (this is automatically added to the datasets).